src/convertToObject.js

```diff
@@ -3,16 +3,32 @@
 /**
  * Implement convertToObject function:
  *
- * Function takes a string with styles (see an example in [stylesString.js](./stylesString.js))
+ * Function takes a string with styles
+ *  (see an example in [stylesString.js](./stylesString.js))
  * and returns an object where CSS properties are keys
- * and values are the values of related CSS properties (see an exampl in [test file](./convertToObject.test.js))
+ * and values are the values of related CSS properties
+ * (see an exampl in [test file](./convertToObject.test.js))
  *
  * @param {string} sourceString
  *
  * @return {object}
  */
 function convertToObject(sourceString) {
-  // write your code here
+  const stylesArray = sourceString
+    .split(';')
+    .map(el => el.trim())
+    .filter(el => el.length > 0);
+
+  const stylesObj = {};
+
+  stylesArray.map(style => {
```

Comment:
good logic 👍 
i'd point out few things although:
- `.map` method returns a new array, so usually it's used when you actually need a new array, otherwise if you just iterate through an array and don't need a new one - better go with `.forEach` - it's an actual method purpose)

Reply:
and essentially i think you would benefit from `.reduce` method that would save you lines of code:
```
  return stylesArray.reduce((stylesObj, style) => {
    const [styleKey, styleValue] = style.split(':')
      .map(piece => piece.trim());

    stylesObj[styleKey] = styleValue;

		return stylesObj;
  }, {})
```